pkg/log/zap/zap.go

```diff
@@ -98,6 +100,17 @@ func Encoder(encoder zapcore.Encoder) func(o *Options) {
 	}
 }
 
+// TimeEncoder configures how the logger will encode time format.
+// See Options.EncoderConfigOptions
+func TimeEncoder(timeEncoder zapcore.TimeEncoder) func(o *Options) {
```

Comment:
@joelanford Now that all [EncoderConfig options](https://godoc.org/go.uber.org/zap/zapcore#EncoderConfig) are configurable I'm thinking we could add helpers for all the other fields while we're here. 
Should be straightforward enough now, e.g:
```Go
func LevelEncoder(levelEncoder zapcore.LevelEncoder) func(o *Options) {
	return func(o *Options) {
		f := func(ec *zapcore.EncoderConfig) {
			ec.EncodeLevel = levelEncoder
		}
		o.EncoderConfigOption = append(o.EncoderConfigOption, f)
	}
}
```
Or wait until someone actually needs them, like the TimeEncoder?

Reply:
As discussed, We would not be adding helpers as of yet.